addons/field_rations/XEH_preInit.sqf

```diff
@@ -18,4 +18,11 @@ GVAR(waterSourceP3Ds) = _cache select 0;
 // List of refill action offsets corresponding to the p3ds in the array above
 GVAR(waterSourceOffsets) = _cache select 1;
 
+// ContextMenuOption
+["ACE_fieldRationsItems", ["CONTAINER"], "Eat/Drink", [], QPATHTOF(ui\icon_survival.paa), [{GVAR(enabled)}, {GVAR(enabled)}], {
```

Comment:
Move to `XEH_postInit.sqf`, add only if `field_rations` is enabled.

Reply:
the entry is only shown if it is enabled
i moved it to postInit, is there a special reason for that?